raiden/network/rpc/client.py

```diff
@@ -509,18 +509,18 @@ def __init__(
 
         result = jsonrpc_client.call(
             'eth_getCode',
-            address_encoder(token_address),
+            token_address,
```

Comment:
Wait wait ... you are giving a binary address into a jsonrpc call here. Why did you make this change?